compiler/luci-interpreter/src/kernels/Conv2D.cpp

```diff
@@ -47,32 +47,38 @@ void Conv2D::configure()
   // We only support (1) and (3) for now.
   if (input()->element_type() == DataType::FLOAT32 && filter()->element_type() == DataType::FLOAT32)
   {
-    assert(bias() == nullptr || bias()->element_type() == DataType::FLOAT32);
+    throwIfConditionFalse(bias() == nullptr || bias()->element_type() == DataType::FLOAT32,
+                          "Input and bias type must be (Float, Float) or (Uint8, Int32)");
```

Comment:
The comment does not match the actual check.

Reply:
this message will be removed if commented above implement. Thanks!